services/app-api/src/resolvers/updateHealthPlanFormData.ts

```diff
@@ -188,6 +191,23 @@ export function updateHealthPlanFormDataResolver(
 
         const editableRevision = planPackage.revisions[0]
 
+        // If CONTRACT_ONLY submission has supporting documents with RATES_RELATED category we convert this to CONTRACT_RELATED.
+        // This was done on the front end in FileUpload.tsx, but we are doing this here to convert these documents when
+        // the submission type has changed in order to save this across the forms because specifically the review and
+        // submit page, will not display documents that are RATE_RELATED under Contract Details sections when changing
+        // submission type to CONTRACT_ONLY.
+        if (
+            unlockedFormData.submissionType === 'CONTRACT_ONLY' &&
+            unlockedFormData.documents.some((document) =>
+                document.documentCategories.includes('RATES_RELATED')
```

Comment:
what if document categories include RATES ?

Reply:
What you said is correct. I forgot about what is happening on submit with this workflow.